docs/docker.md

```diff
@@ -0,0 +1,52 @@
+# Activate cloud orchestrator in the remote server
+
+This page describes how to run cloud orchestrator managing docker instances
+containing Host Orchestrator inside.
+
+## Prepare docker image
+
+Please follow the docker part of
+[README.md](https://github.com/google/android-cuttlefish/blob/main/README.md#docker)
+in `google/android-cuttlefish` github repository, and check if the docker image
+`cuttlefish-orchestration` exists.
+
+## Build cloud orchestrator
+
+To build cloud orchestrator, please execute:
+```bash
+git clone https://github.com/google/cloud-android-orchestration.git
+cd cloud-android-orchestration # Root directory of git repository
+scripts/docker/build.sh
+```
+
+If build was successful, you would be able to see cloud_orchestrator binary at
+`/path/to/cloud-android-orchestration/cloud_orchestrator`.
```

Comment:
should this be changed to `cloud_orchestrator`?

Reply:
Since leaving `cloud_orchestrator` only doesn't seem like the path, so I rewrote the sentence.